tools/nhlt-parse.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2020 Intel Corporation. All rights reserved.
+#
+# usage: nhlt-parse.py <nhtl-dump>
+#
+# examples:
+#   cat /sys/firmware/acpi/tables/NHLT >nhtl.bin
+#   ./nhtl-parse.py nhtl.bin
+#
+# references:
+#  - https://01.org/sites/default/files/595976_intel_sst_nhlt.pdf
+
+from collections import namedtuple
+import struct
+import sys
+
+if len(sys.argv) == 1:
+    infile = '/sys/firmware/acpi/tables/NHLT'
+else:
+    infile = sys.argv[1]
+
+f = open(infile, 'rb')
```

Comment:
Do we need some try catch here to avoid wrong path used.

Reply:
I don't think so, Python has very good error messages by default.